consensus/quorum/quorum.go

```diff
@@ -124,6 +125,7 @@ type Decider interface {
 	IsQuorumAchieved(Phase) bool
 	IsQuorumAchievedByMask(mask *bls_cosi.Mask) bool
 	QuorumThreshold() numeric.Dec
+	OneThirdQuorum() numeric.Dec
```

Comment:
can remove this interface if we won't use it to avoid confusion.

Reply:
yes